sdk/signalr/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+__all__ = ["WebPubSubServiceClient"]
+
+from typing import TYPE_CHECKING
+from copy import deepcopy
+
+import jwt
+
+import azure.core.pipeline as corepipeline
+import azure.core.pipeline.policies as corepolicies
+import azure.core.pipeline.transport as coretransport
+
+# Temporary location for types that eventually graduate to Azure Core
+from .core import rest as corerest
+
+from ._policies import JwtCredentialPolicy
+
+if TYPE_CHECKING:
+    import azure.core.credentials as corecredentials
+    from azure.core.pipeline.policies import HTTPPolicy, SansIOHTTPPolicy
+    from typing import Any, List, cast
+
+
+class WebPubSubServiceClient(object):
+    def __init__(self, endpoint, credential, **kwargs):
+        # type: (str, corecredentials.AzureKeyCredential, Any) -> None
+        """Create a new WebPubSubServiceClient instance
+
+        :param endpoint: Endpoint to connect to.
+        :type endpoint: ~str
+        :param credential: Credentials to use to connect to endpoint.
+        :type credential: ~azure.core.credentials.AzureKeyCredential
+        :keyword api_version: Api version to use when communicating with the service.
+        :type api_version: str
+        :keyword user: User to connect as. Optional.
+        :type user: ~str
+        """
+        self.endpoint = endpoint.rstrip("/")
+        transport = kwargs.pop("transport", None) or coretransport.RequestsTransport(
+            **kwargs
+        )
+        policies = [
+            corepolicies.HeadersPolicy(**kwargs),
+            corepolicies.UserAgentPolicy(**kwargs),
+            corepolicies.AsyncRetryPolicy(**kwargs),
+            corepolicies.ProxyPolicy(**kwargs),
+            corepolicies.CustomHookPolicy(**kwargs),
+            corepolicies.AsyncRedirectPolicy(**kwargs),
+            JwtCredentialPolicy(credential, kwargs.get("user", None)),
```

Comment:
You added `corepolicies.ContentDecodePolicy(**kwargs),` to the sync pipeline so I guess it should be here too?

Reply:
For this service, we actually don't need it. Will remove instead...